src/components/selectable-dropdown/SelectableDropdown.jsx

```diff
@@ -0,0 +1,55 @@
+import { Dropdown } from '@edx/paragon';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+
+function SelectableDropdown({
```

Comment:
In Paragon (https://edx.github.io/paragon/components/dropdown) is just a pass-through of the React Bootstrap component.

Using https://react-bootstrap.github.io/components/dropdowns/#custom-dropdown-components would be a better match for this ui.

Reply:
@giovannicimolin

I'm not quite sure I follow you... The `SelectableDropdown` component uses the React Bootstrap `Dropdown`, `Dropdown.Toggle`, `Dropdown.Menu` and `Dropdown.Item` components that are passed through. I tend to think the UI for the dropdown elements is rendered well (when comparing to the mockups). What exactly do think doesn't look right? How should the custom dropdown components be used to improve the look?

Reply:
I think this component can probably be lifted up to Paragon if it's used enough, for now we can just use it as-is.

Reply:
I don't think it needs to be lifted up to Paragon, as it provides enough flexibility to match the mockups as is. The link I've posted above has an example that is pretty close to the desired result (it's a combination of an `a` tag + Dropdown menu and only uses bootstrap classes).